src/SymbolManager.cpp

```diff
@@ -108,17 +109,49 @@ void SymbolManager::refresh()
  * File list support functions
  */
 
+enum TableColumns {
+	FILENAME = 0,
+	DESTINATION_SLOT,
+	DESTINATION_SEGMENT,
```

Comment:
Is this still work in progress? I don't see `DESTINATION_SEGMENT` being used anywhere.